tutorial/lesson_10_aot_compilation_generate.cpp

```diff
@@ -72,10 +72,10 @@ int main(int argc, char **argv) {
     // arguments to the routine. This routine takes two. Arguments are
     // usually Params or ImageParams.
     std::vector<Argument> args = {input, offset};
-    brighter.compile_to_static_library("lesson_10_halide", args);
+    brighter.compile_to_static_library("lesson_10_halide", args, "lesson_10_halide");
```

Comment:
I think we should pick a function name distinct from the filename here to demonstrate that they're different things. Maybe an explicit "brighter"?